web/src/main/java/org/springframework/security/web/header/HeaderWriterFilter.java

```diff
@@ -38,55 +39,55 @@
  *
  * @author Marten Deinum
  * @author Josh Cummings
+ * @author Ankur Pathak
  * @since 3.2
- *
  */
 public class HeaderWriterFilter extends OncePerRequestFilter {
 
+
 	/**
-	 * Collection of {@link HeaderWriter} instances to write out the headers to the
+	 * Composite{@link CompositeHeaderWriter} containing Collection of {@link HeaderWriter} instances to write out the headers to the
 	 * response.
 	 */
-	private final List<HeaderWriter> headerWriters;
+	private final CompositeHeaderWriter compositeHeaderWriter;
```

Comment:
Let's code to the interface. So we'd have this be a `HeaderWriter headerWriter`, for example. Probably update the Javadoc accordingly, as well.

Reply:
Done.